android/src/main/java/com/reactlibrary/RNMailCoreModule.java

```diff
@@ -85,6 +86,77 @@ public void failed(MailException e) {
         });
       }
     });
+  }
+
+  @ReactMethod
+  public void loginImap(final ReadableMap obj, final Promise promise){
+    getCurrentActivity().runOnUiThread(new Runnable() {
+      @Override
+      public void run() {
+        UserCredential user = new UserCredential(obj);
+        mailClient.initIMAPSession(user,promise);
+      }
+    });
+  }
+
+  @ReactMethod
+  public void loginSmtp(final ReadableMap obj, final Promise promise){
+    getCurrentActivity().runOnUiThread(new Runnable() {
+      @Override
+      public void run() {
+        UserCredential user = new UserCredential(obj);
+        mailClient.initSMTPSession(user,promise);
+      }
+    });
+  }
+
+  @ReactMethod
+  public void sendMail(final ReadableMap obj, final Promise promise){
```

Comment:
Are there 2 methods with the name and signature ?

Reply:
Yes, I forgot to delete the method, because I duplicated it in another class and I am modifying it there

Reply:
we should delete the one that does not have functionality